ecs-logs-collector.sh

```diff
@@ -93,7 +92,7 @@ parse_options() {
         help && exit 0
         ;;
       *)
-        echo "Command not found: '--$param'"
+        echo "Command not found: '$param'"
```

Comment:
Can we change "command" to "parameter" here. Would be more informative.

Reply:
I agree with you 😃